src/assets/toolkit/scripts/home-animation.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * GSAP is powerful and performant, but its organization is a little old-school.
+ * This imports a bunch of functions to the global scope.
+ *
+ * See: https://github.com/greensock/GreenSock-JS/issues/71
+ */
+
+import TweenLite from 'gsap/src/uncompressed/TweenLite';
+import TimelineMax from 'gsap/src/uncompressed/TimelineMax';
+import EasePack from 'gsap/src/uncompressed/easing/EasePack';
+import CSSPlugin from 'gsap/src/uncompressed/plugins/CSSPlugin';
+import AttrPlugin from 'gsap/src/uncompressed/plugins/AttrPlugin';
+
+/**
+ * Settings
+ */
+
+const settings = {
+  morph: {
+    delay: '+=1',
+    duration: 0.5,
+    ease: Elastic.easeOut.config(0.5, 0.4)
+  }
+};
+
+/**
+ * Timelines
+ */
+
+const timelines = {
+  morph: new TimelineMax({
+    repeat: -1
+  })
+};
+
+/**
+ * Elements
+ */
+
+const dom = (function (nodeList) {
+  const result = {};
+
+  for (let i = 0; i < nodeList.length; i++) {
+    let node = nodeList[i];
+    let id = node.getAttribute('id');
+
+    if (id) {
+      result[id] = node;
+    }
+  }
+
+  return result;
+})(document.querySelectorAll('*[id]'));
```

Comment:
But remember, this is in _a separate document_. So we don't have _any_ polyfills initialized in the HTML page. This script is loaded by and only exists in the SVG document, which is embedded via `<object>`.

I don't think it's worth including an additional dependency within the SVG document to save 8 lines of JavaScript. But maybe I'm wrong about that.